packages/data-stores/src/plans/test/resolvers.ts

```diff
@@ -20,15 +20,15 @@ describe( 'getSupportedPlans', () => {
 				// and rounds to 0 decimal places
 				currency_code: 'INR',
 				product_slug: PLAN_FREE,
-				raw_price: 13,
+				raw_price: 0,
```

Comment:
We should remove the mention of the decimal places from the comment (a few lines above), now that this comes from an external package.

Maybe the comment should change to something like "Free plan, with a cost of 0"